common/websocket/index.ts

```diff
@@ -65,7 +65,9 @@ class WebSocketService {
 
     private removeConnection(userId: string, connectionId: string) {
         if (!this.connectedUsers.has(userId) || !this.connectedUsers.get(userId)?.has(connectionId)) {
-            log.error(`UserId or clientId not found in connected clients: ${[...this.connectedUsers.entries()]}.`);
+            log.error('UserId or clientId not found in connected clients.', new Error('UserId or clientId not found in connected clients.'), {
+                connectedUsers: [...this.connectedUsers.entries()],
```

Comment:
Is this needed? Logging all connected users could potentially be heavy

Reply:
Good Point. I'll remove it.